build.cake

```diff
@@ -0,0 +1,115 @@
+// Install tools.
+#tool "nuget:https://www.nuget.org/api/v2?package=GitVersion.CommandLine&version=3.6.2"
+
+///////////////////////////////////////////////////////////////////////////////
+// ARGUMENTS
+///////////////////////////////////////////////////////////////////////////////
+
+var target          = Argument<string>("target", "Default");
+var configuration   = Argument<string>("configuration", "Release");
+
+// Load other scripts.
+#load "./build/parameters.cake"
```

Comment:
@shiftkey yes, it is a requirement. Must be excluded through gitignore. I have it locally on my machine. I will correct later once at a computer. Thanks!